community-operators/elasticsearch-operator/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,218 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: elasticsearch-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Logging & Tracing"
+    certified: "false"
+    description: |-
+      The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for tracing and cluster logging.
+    containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
+    createdAt: 2019-02-20T08:00:00Z
+    support: AOS Cluster Logging, Jaeger
+    alm-examples: |-
+        [
+            {
+                "apiVersion": "logging.openshift.io/v1alpha1",
+                "kind": "Elasticsearch",
+                "metadata": {
+                  "name": "elasticsearch"
+                },
+                "spec": {
+                  "managementState": "Managed",
+                  "nodeSpec": {
+                    "image": "quay.io/openshift/origin-logging-elasticsearch5:latest",
+                    "resources": {
+                      "limits": {
+                        "memory": "1Gi"
+                      },
+                      "requests": {
+                        "memory": "512Mi"
+                      }
+                    }
+                  },
+                  "nodes": [
+                    {
+                        "nodeCount": 1,
+                        "roles": ["client","data","master"], 
+                        "redundancyPolicy": "SingleRedundancy"
+                    }
+                  ]
+                }
+            }
+        ]
+spec:
+  displayName: Elasticsearch Operator
+
+  description: |
+    The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
+    This operator only supports OKD Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
+    being used as a general purpose manager of Elasticsearch clusters running on OKD.
+
+    Once installed, the operator provides the following features:
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+
+  keywords: ['elasticsearch', 'jaeger']
+
+  maturity: Seamless Upgrades
```

Comment:
This should be removed, see #92

Reply:
@SamiSousa I see this value goes into an annotation but what should 'maturity' be set to?  Does it need to exist at all?

Reply:
It need not exist. I'm not exactly sure what OLM does with this value, but I do know that the CSV fails to install if the value isn't set right.

Reply:
@alecmerdler Any reason not to remove this field from a CSV?

Reply:
OLM doesn't use this anywhere.

Reply:
OLM defines this field in it's CSV docs: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata
Also the catalog-operator checks this field, and fails if the value is set to something unexpected. But it doesn't fail if the field isn't set.

Reply:
@jcantrill You can move "Seamless Upgrades" to `metadata.annotations.capabilities` since that will be used in OperatorHub.io

Reply:
Updated to move to an annotation